docs/gds-supported-platform/Updating EKS.md

```diff
@@ -0,0 +1,14 @@
+# Updating EKS
+
+* Update EKS in sandbox
+* Update nodes in sandbox
+
+If no issues are witnessed with sandbox, promote the EKS update to verify/portfolio.
+
+Following a period of stable running without errors connected to the update, promote the node update in verify/portfolio.
+
+## Todo
+What about plugins? Core DNS, AWS VPC CNI, kube-proxy.
```

Comment:
I think the aws documentation (you link to below) covers what needs to be done with these components, so we don't need to call them out seperately

Reply:
ok
corrected.